.env.development

```diff
@@ -11,6 +11,8 @@ AWS_SECRET_ACCESS_KEY=fsdlrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
 AWS_BUCKET_NAME=fsd-bucket
 AWS_ENDPOINT_OVERRIDE=http://localhost:4566
 AWS_REGION=eu-west-2
-AWS_SQS_QUEUE_NAME=fsd-queue
-AWS_DLQ_QUEUE_NAME=fsd-dlq
+AWS_SQS_IMPORT_APP_QUEUE_NAME=import-queue
+AWS_DLQ_IMPORT_APP_QUEUE_NAME=import-dlq
```

Comment:
Very nice! I noticed they weren't very specific, this will make future work easier ⭐